zoo-css-challenge/style.css

```
@@ -75,55 +103,142 @@ footer {
   justify-content: space-between;
 }
 
+/*Introduction*/
+
 .introduction {
+  background-color: #fff;
 }
 
+/*Badge*/
+
+/* 9) Badge: As the position property was set to fixed, by replacing the 
+      "bottom" property with "top" the badge moved to the desired position. */
 #badge {
   height: 120px;
   width: 120px;
   background-color: #bbb;
   border-radius: 50%;
   display: inline-block;
   position: fixed;
-  bottom: 20px;
+  top: 20px;
   right: 20px;
-
   display: flex;
   justify-content: center;
   align-content: center;
   flex-direction: column;
 }
 
+/* 9) Programs: Added a transition time of 0.8 seconds with ease-in to smooth out 
+      colour change on hover */
 #badge:hover {
   background-color: yellow;
+  transition: ease-in 0.8s;
```

```suggestion
  transition: background-color 0.8s ease-in;
```
It's good practise to specify the CSS property we want to animate, for performance reasons. Also, we should provide the values in this order according to the documentation

https://developer.mozilla.org/en-US/docs/Web/CSS/transition

---

Thanks for pointing this out, to be honest, I did not check the documentation as I was already familiar with the transition property, so I learnt a valuable lesson here, always check the documentation :)